internal/network/manifests_generator.go

```diff
@@ -238,6 +239,126 @@ func (m *ManifestsGenerator) AddSchedulableMastersManifest(ctx context.Context,
 	return nil
 }
 
+const (
+	diskEncryptionManifest = `apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  name: {{.NAME}}
+  labels:
+    machineconfiguration.openshift.io/role: {{.ROLE}}
+spec:
+  config:
+    ignition:
+      version: 3.2.0
+    storage:
+      luks:
+        - name: root
+          device: /dev/disk/by-partlabel/root
+          clevis:
+		  {{ if .ENCRYPTION_MODE_TPMV2 }}
+            tpm2: true
+		  {{ end }}
+		  {{ if .ENCRYPTION_MODE_TANG }}
```

Comment:
Sure this is the right indentation?

Reply:
I think that github show you a wrong indention - this is not how it is in the code